tests/testthat/test-get_pvol_se.R

```diff
@@ -1,6 +1,12 @@
 test_that("Pvol for Sweden can be downloaded", {
   skip_if_offline("opendata-download-radar.smhi.se")
   time <- as.POSIXct(Sys.time(), tz = "Europe/Helsinki") - lubridate::hours(10)
+  skip_if_not(
```

Comment:
A helper for skipping might be useful, do you think this could be generalised to other tests?